### ISSUE TITLE
Bridge not easily accessible / installable

### ISSUE THREAD

**Issue by andibuch** [yuliskov/SmartTube#505]
I'm a longterm user of SmartYouTubeTV / SmartTubeTV

I really appreciated the feature you incorporated into an old release of SmartYouTubeTV, where it would prompt new installs to install the bridge-app that allowed your FireTV to recognize SmartYouTubeTV as being YouTube itself / send voice searches for YouTube there / launch with the YouTube icon / etc.

When I installed SmartTubeTV, the old bridge updated to redirect to SmartTubeTV which was great :)

However, on a friend's fire stick, I had an issue a while back where after installing / uninstalling / reinstalling SmartYouTubeTV, the bridge app disappeared – as did any notification to install it

Moreover, when more recently I uninstalled SmartYouTubeTV altogether, and installed SmartTubeTV from scratch, I was never given an option to install the bridge app – and thus, the FireTV refused to recognize SmartTubeTV as YouTube

I was at a loss as to what to do, as the bridge apk has no public listing that I could find – be it in the releases for SmartTubeTV, SmartYouTubeTV, or anywhere else on the internet

HOWEVER, after digging through the issues of SmartYouTubeTV, I finally found this issue: https://github.com/yuliskov/SmartYouTubeTV/issues/697

In it, you directly link to the bridge app – https://github.com/yuliskov/SmartYouTubeTV/releases/download/stable/Amazon_SYTV_Bridge.apk

So I was able to download that, install it onto the FireTV, and viola – it worked perfect!

So... basically, I recommend making a public listing for that Amazon_SYTV_Bridge.apk – be it here on this github in the releases, or elsewhere – and / or giving a menu-option in-app to install said .apk within SmartTubeTV

Thank you for your consideration!

**Comment by finnzz** [comment]
Thank you for the recommendation!  Just one suggestion, the new app is called SmartTubeNext.  It was a little hard following your post, but I got the central message lol

**Comment by fancellu** [comment]
The above url is now broken

**Comment by bs-github** [comment]
Unfortunately the latest available bridge apk that I found installs on the FireTV 4k stick, but fails to run with an error message that tells me to get an updated version.

Is there any chance to get a new version (probably just a recompiled one)?

**Comment by finnzz** [comment]
Do you have the original YouTube app installed? If so you need to uninstall it.

I'm using the same bridge app that came out a year ago without any problem on FireOS7.  There are 2 bridge apps, one for FireOS and one for AndroidTV.  Make sure you are using the FireOS version.
https://kutt.it/stn_bridge_amazon

**Comment by bs-github** [comment]
- I have deleted the original YouTube App.
- I have installed the **Amazon_SYTV_Bridge.apk Version 13.92** downloaded from https://kutt.it/stn_bridge_amazon on the 14th of January 2022
- The Fire TV Stick 4K runs **Fire OS 6.2.8.1 (NS6281/4061)**

But I get the error message which tells me to look for a newer version anyways.
`Die installierte Version von Amazon Smart YouTube TV
Bridge kann nicht geöffnet werden. Bitte überprüfen
Sie, ob ein Update in Ihrer Bibliothek verfügbar ist.`

**Comment by finnzz** [comment]
I don't think FireOS6 should be a problem.  The bridge app just uses the same package name as the official YouTube app, and upon opening it, redirects you to the SmartTubeNext app.

Just to confirm, you do have the SmartTubeNext app and the Amazon bridge app both installed?

The error message makes me think the bridge app may not be able to find the SmartTubeNext app.

**Comment by bs-github** [comment]
Yes, I do have the STN app (Version 13.92) installed and the app itself works flawlessly :-).
Just the bridge app does not.

**Comment by finnzz** [comment]
> Yes, I do have the STN app (Version 13.92) installed and the app itself works flawlessly :-). Just the bridge app does not.

Ok, unfortunately I don't have a device with FireOS6 to confirm whether the bridge app works or not.

It may be worth asking in the FireTV community on Reddit if the bridge app works on FireOS6. You'll get a fast response there:
https://reddit.com/r/firetvstick 
https://reddit.com/r/firetv

Looking at the Amazon bridge app from the link above, the AndroidManafest.xml does reference:
com.liskovsoft.smartyoutubetv (the old SmartYouTubeTV app)

You don't have SmartYouTubeTV and SmartTubeNext both installed right? If so, try uninstalling SmartYouTubeTV.

I remember that even though the bridge points to the old app package name (com.liskovsoft.smartyoutubetv), that it also worked with the new package name (com.liskovsoft.smarttubetv.beta). Maybe on FireOS the old reference to the old package name is a problem?  You could open the AndroidManafest.xml in the APK and change the old package name to the new one, install the APK, and see if that fixes the problem.

**Comment by finnzz** [comment]
I edited out the old package name with the new one of you want to give that a try
https://sharefast.me/QV

**Comment by bs-github** [comment]
Thank you very much for your efforts to help me!
I've tried the version linked above. 
Unfortunately that did not make any difference. :-(

**Comment by finnzz** [comment]
Sorry, it was probably wishful thinking that would be enough to fix it.
